source/renderer/app/features/discreet-mode/feature.js

```diff
@@ -50,4 +51,20 @@ export class DiscreetMode extends Feature {
       this.openInDiscreetMode = nextSetting;
     });
   };
+
+  hideSensitiveData(data: string) {
+    if (!this.isDiscreetMode) {
+      return data;
+    }
+
+    const content = data.split(' ');
+
+    if (content.length <= 1) {
+      return SENSITIVE_DATA_SYMBOL;
+    }
+
+    const tickerSymbol = content.pop();
+
+    return `${SENSITIVE_DATA_SYMBOL} ${tickerSymbol}`;
```

Comment:
Let's make this more generic, this is 100% coupled to ticker values and even their "markup" like that the ticker symbol always is at a specific position in the string etc.

Reply:
Removing currencyValue from `hideSensitiveData` will result in a lot of duplicated logic like:

```
let formattedValue =   formattedTokenWalletAmount(amount, null, 0)

if(discreetModeFeature.isDiscreetMode) {
       const hiddenValue = discreetModeFeature.hideSensitiveData(
                  formattedTokenWalletAmount(amount, null, 0)
                )

const currencyValue = metadata?.token || ''

formattedValue = [hiddenValue, currencyValue].join(' ').trim()
}                
```

Of course this could be moved to `formatters.js` or similar.